.eslintcache

```diff
@@ -0,0 +1 @@
+[{"/Users/pharrington/Code/colonyDiscord/src/events.ts":"1"},{"size":10064,"mtime":1653971407051}]
```

Comment:
Please add the `.eslintcache` file in another line of the `.gitignore` file. It seems to be something that VSCode creates, so that's why it got in here I assume.